pkg/selector/pod/selector.go

```diff
@@ -313,6 +308,7 @@ func listPods(ctx context.Context, c client.Client, r client.Reader, spec v1alph
 
 	filterPods := make([]v1.Pod, 0, len(pods))
 	for _, pod := range pods {
+		pod := pod
```

Comment:
how about using a new name?

Reply:
I think it's a proper way to avoid `implicit memory aliasing in for loop` error, and I referenced this way from the code of ChaosMesh actually. For example: https://github.com/chaos-mesh/chaos-mesh/blob/master/pkg/time/time_skew_amd64.go#L104